be/src/storage/size_tiered_compaction_policy.cpp

```diff
@@ -27,8 +27,13 @@ namespace starrocks {
 
 SizeTieredCompactionPolicy::SizeTieredCompactionPolicy(Tablet* tablet) : _tablet(tablet) {
     _compaction_type = INVALID_COMPACTION;
-    _max_level_size =
-            config::size_tiered_min_level_size * pow(config::size_tiered_level_multiple, config::size_tiered_level_num);
+    if (_tablet->keys_type() == KeysType::DUP_KEYS) {
+        _level_multiple = std::max(config::size_tiered_level_multiple_dupkey, config::size_tiered_level_multiple);
+    } else {
+        _level_multiple = config::size_tiered_level_multiple;
```

Comment:
The default value `config::size_tiered_level_multiple_dupkey` is only two times of `config::size_tiered_level_multiple`. How much compaction will be triggered because of the gap?

Reply:
(n/5 + n/5^2 + n/5^3 + n/5^4)/(n/10 + n/10^2 + n/10^3 + n/5^4)

Reply:
Why not use `2 * config::size_tiered_level_multiple` instead of a specific configuration?

Reply:
duplicate key is quite difference with agg/unique key, so that i add a special config for it.